picky/src/key.rs

```diff
@@ -116,6 +116,52 @@ impl TryFrom<&'_ PrivateKey> for RsaPublicKey {
 }
 
 impl PrivateKey {
+    pub fn from_components(
```

Comment:
We might support other type of keys.

```suggestion
    pub fn from_rsa_components(
```